new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/LoggingUtil.java

```diff
@@ -137,4 +232,162 @@ private static void writeToFile(Object obj, String resultDir, String fileName) t
     }
   }
 
-}
+  /**
+   * Copy files from persistent volume to local folder.
+   * @param namespace name of the namespace
```

Comment:
pv's are not namespaced - explain what the namespace param is for more clearly

Reply:
modified the descriptions to be more clear.
For the record
The namespace is used for creating the temporary pod not for pv.